src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala

```diff
@@ -17,19 +17,19 @@ object WsClient extends LazyLogging {
   implicit val bodyWrites: BodyWritable[JsValue] =
     BodyWritable(a => InMemoryBody(ByteString.fromArrayUnsafe(Json.toBytes(a))), "application/json")
   private val asyncClient: StandaloneAhcWSClient = {
-    implicit val system: ActorSystem = ActorSystem()
+    implicit val system: ActorSystem             = ActorSystem()
     implicit val materializer: ActorMaterializer = ActorMaterializer()
 
     StandaloneAhcWSClient()
   }
 
   def get(
-           uri: String,
-           queryParameters: Map[String, String] = Map.empty[String, String],
-           headers: Map[String, String] = Map.empty[String, String],
-           cookies: Seq[WSCookie] = Seq.empty[WSCookie],
-           followRedirects: Boolean = false
-         ): StandaloneWSResponse = {
+    uri: String,
+    queryParameters: Map[String, String] = Map.empty[String, String],
+    headers: Map[String, String] = Map.empty[String, String],
+    cookies: Seq[WSCookie] = Seq.empty[WSCookie],
+    followRedirects: Boolean = false
+  ): StandaloneWSResponse = {
```

Comment:
Should we syncronise auto formatting

Reply:
i think the repo stand is sbt test:scalafmt